viewing_party/party.py

```diff
@@ -1,23 +1,162 @@
 # ------------- WAVE 1 --------------------
+from platform import libc_ver
+
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title != None:
+        new_movie["title"] = title
+        if genre != None:
+            new_movie["genre"] = genre
+            if rating != None:
+                new_movie["rating"] = rating
+                return new_movie
+            else:
+                return None
+        else:
+            return None
+    else:
+        return None

```

Comment:
This code works, but in general if you find yourself nesting code so deeply, it's likely a sign that there is a better way to arrange the code. About more than 3 levels of nesting becomes hard to read.

Towards that end, I would consider moving them to check the input arguments before you go ahead and initialize `new_movie`. That way, if any are `None`, you can avoid the allocation of `new_movie`.

Additionally, you can definitely get this down to one if statement either by using `and` or putting them into a temporary list like this:

```python
if None in [title, genre, rating]:
    return None
```

Now, while this is more succinct, it could easily be argued that separate checks are more explicit. They also would make it easier to expand for other checks of the individual values:

```python
if rating == None or rating < 0 or rating > 5:
    return None
```

But this is a rather minor issue.